grant-project/grant-monorepo
# Markdown & MarkdownEditor to less. Remove styled-components SSR/dependencies.

Closes #41.

### What this does
- convert `Markdown` & `MarkdownEditor` styles to less
- remove styled-components SSR code
- remove styled-components & related dependencies

### Testing
- update deps, `yarn`
- make sure `yarn dev` and `yarn build` run clean
- make sure create>details (preview) looks correct
![screen shot 2018-09-22 at 1 11 44 pm](https://user-images.githubusercontent.com/11430954/45920410-34b63500-be69-11e8-92d5-612c59005bb5.png)
- make sure create>review details render correctly
![screen shot 2018-09-22 at 1 13 27 pm](https://user-images.githubusercontent.com/11430954/45920418-63341000-be69-11e8-9cf6-8d935f643cf6.png)
- make sure proposal>comments (preview) looks correct
![screen shot 2018-09-22 at 1 16 49 pm](https://user-images.githubusercontent.com/11430954/45920499-582daf80-be6a-11e8-8984-efc57caad534.png)
